pkg/awsManager/aws_manager_test.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/golang/mock/gomock"
 	"github.com/openshift/aws-account-shredder/pkg/localMetrics"
 	"github.com/openshift/aws-account-shredder/pkg/mock"
-	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
```

Comment:
Did you verify that this change didn't break the current API?

Reply:
it does pass the unit tests

Reply:
just ran the shredder and no errors